sdlc_xmalab.py

```diff
@@ -996,7 +740,9 @@ def analyze_marker_similarity_trial(working_dir):
     trial2_path = os.path.join(f'{working_dir}/trials', project['trial_2_name'])
     
     # Get a list of markers that each trial have in commmon
-    markers_in_common = [marker for marker in get_bodyparts_from_xma(trial1_path, mode='rgb', split_markers=False, crossed_markers=False) if marker in get_bodyparts_from_xma(trial2_path, mode='rgb', split_markers=False, crossed_markers=False)]
+    bodyparts1 = get_bodyparts_from_xma(project, trial1_path, split_markers=False, crossed_markers=False)
```

Comment:
NOTE: From the tests, the marker similarity functions are failing with key errors because this needs to be run in RGB mode all the time (adding_cam1 and _cam2 to 2D/per_cam marker names (F1_cam1_X vs. F1_X).

Reply:
Going to be thinking about this one over the next couple of days, let me know if you have any ideas in the meantime for solving @vytran17. Simplest way is going to be keeping things as-is, however I don't like having to specify 'rgb' as the flag when I want marker names as they would be in the CSV itself.

Reply:
switched the body parts function back to take mode as a parameter (same as it was before). but i did change it to not have a default so that callers specify it explicitly so it's less confusing.  tests are now passing